universe/api/middleware/verify_user.go

```diff
@@ -14,10 +14,9 @@ func VerifyUser(log *zap.SugaredLogger) gin.HandlerFunc {
 	return func(c *gin.Context) {
 		token, err := api.VerifyToken(c, api.GetTokenFromRequest(c))
 		if err != nil {
-			log.Debug(errors.WithMessage(err, "Middleware: VerifyUser: failed to verify token"))
-			c.AbortWithStatusJSON(http.StatusForbidden, gin.H{
-				"message": "failed to verify access token",
-			})
+			err = errors.WithMessage(err, "Middleware: VerifyUser: failed to verify token")
+			log.Debug(err)
+			api.AbortRequest(c, http.StatusForbidden, "failed_to_verify_access_token", err)
```

Comment:
whats the purpose of using AbortRequest, if you additionally doing some more things?

Reply:
The purpose of AbortRequest to make code shorter (DRY)